src/NuGet.Core/NuGet.ProjectModel/Utf8JsonStreamReader.cs

```diff
@@ -180,9 +180,12 @@ internal IList<string> ReadStringArrayAsIList(IList<string> strings = null)
                 {
                     string value = _reader.ReadTokenAsString();
 
-                    strings ??= new List<string>();
+                    if (value != null)
+                    {
+                        strings ??= new List<string>();
 
```

Comment:
nit: do we need the white line?

Reply:
not sure I follow what you're referring to.

Reply:
Line 186, but it was not added in this PR.